tests/core20/snapd-failover/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Check that snapd failure handling works
```

Comment:
I don't see anything specific to uc20 in this, can we just disable the main one for uc20 specifically instead of making a duplicate test?

Reply:
You are right, it is duplicated from core18 test suite. So far it is disabled for core20, and should be enabled soon. The problem that the test suite core18 runs just on core18 systems, so to run this test on core20 we need to have a copy on core20 test suite.
Perhaps we could move the test to main suite to avoid have it duplicated, what do you think? @mvo5 any thoughts?

Reply:
we should think how to organize our suites,  we have core only tests in main and new core18 and core20 tests, but actually there are some tests that are 18+ tests

Reply:
also the name of the suites is confusing, because core20 is more the base than UC20 etc